### PR TITLE
Adding documentation for Roku hub and remote

### PR DESCRIPTION
**Pull request by alandtse**
**Description:**
Updating documentation to support Roku hub and remote

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#12542

## Checklist:

- [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/roku.markdown [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+The `roku` platform can also be forced to load by adding the following lines to your `configuration.yaml`:
+
+```yaml
+# Example configuration.yaml entry
+roku:
+  host: 192.168.1.100
+  name: Roku
+```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/roku.markdown]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

**Review by frenck — CHANGES_REQUESTED** [review]
@alandtse Thank you for sending this PR in! I've reviewed it, made some minor changed, and left a comment. Could you please take a look at it? 👍

**Comment by frenck** [comment]
Rebased the PR in order to fix the build issues.

@alandtse Friendly reminder, could you please take a look at the open requests? 👍

File: source/_components/roku.markdown (hunk 1 above)
Comment (alandtse) [reply source/_components/roku.markdown]: Done.

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Done, approved. Can be merged as soon as the parent PR gets merged.

**Review by frenck — APPROVED** [review]

**Comment by DubhAd** [comment]
I note that the linked PR was closed 11 days ago, closing this too

**Pull request closed without merging**